iohub/cli/cli.py

```diff
@@ -73,13 +73,22 @@ def info(files, verbose):
     is_flag=True,
     help="Dump postion labels in MM metadata to Omero metadata",
 )
-def convert(input, output, format, grid_layout, label_positions):
+@click.option(
+    "--scale-voxels",
```

Comment:
Since we decided to with a default of transferring the scale, maybe this option could be something like "--suppress-scale"?

Reply:
I made it a boolean option that defaults to true.